tests/bats/assets.subtheme.custom.bats

```diff
@@ -84,12 +84,6 @@ load _helper
   assert_dir_exists "components_combined"
   assert_dir_exists "components_combined/01-atoms/button"
   assert_file_exists "components_combined/01-atoms/button/button.twig"
-  assert_dir_exists "components_combined/01-atoms/demo-button"
```

Comment:
removed as examples not included 
https://github.com/civictheme/monorepo-drupal/blob/feature/CIVIC-1052/web/themes/contrib/civictheme/civictheme_create_subtheme.php#L573